tests/vitest.config.ts

```diff
@@ -0,0 +1,12 @@
+import { defineConfig } from 'vitest/config'
+import react from '@vitejs/plugin-react';
+
+console.log('loading vitest config');
```

Comment:
Should we keep the log?

Reply:
Personally i like having that log to see if it **really** loads the config files 😅 But i can remove them if we are done moving parts around

Reply:
Let's keep it for now, if it makes too much noise we can remove it later. (I didn't run the PR only review the code)